lib/search/find_events.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Onvif.Search.FindEvents do
+  import SweetXml
+  import XmlBuilder
+  require Logger
+
+  def soap_action, do: "http://www.onvif.org/ver10/search/wsdl/FindEvents"
+
+  def request(device, args) do
+    Onvif.Search.request(device, args, __MODULE__)
+  end
+
+  def request_body([included_recordings, start_point, end_point, search_filter, keep_alive_time]) do
```

Comment:
I think we are treating the parameters differently in a few contexts, so I will pass through and make adjustments to the contracts before releasing